static/app/components/events/highlights/highlightsDataSection.tsx

```diff
@@ -147,6 +148,39 @@ function HighlightsData({
     highlightContext,
     location,
   });
+  const highlightTagItems = getHighlightTagData({event, highlightTags});
+
+  // find the replayId from either context or tags, if it exists
+  const contextReplayItem = highlightContextDataItems.find(
+    e => e.data[0].key === 'replay_id'
+  );
+  const contextReplayId = contextReplayItem?.value ?? '';
+
+  const tagReplayItem = highlightTagItems.find(e => e.originalTag.key === 'replayId');
+  const tagReplayId = tagReplayItem?.value ?? '';
+
+  // if the id doesn't exist for either tag or context, it's rendered as '--'
```

Comment:
to clarify, the contextReplayItem.value and tagReplayItem.value could be '--'?

Reply:
yup, if there isn't a value for it, then that's the default value returned